### ISSUE TITLE
Show ecoMode or rainSensor controls only for devices that support it

### ISSUE THREAD

**Issue by the-smart-home-maker** [jonkristian/alko#11]
Not all Robolinho devices are equipped with a rain sensor or an eco mode. So the integration should leverage the information provided in the pimInfo (here it is stated whether the device provide rain sensor and eco mode or not) to selectively show these controls only of they are supported by the device.

**Comment by jonkristian** [comment]
Good point. Should be fixed in [v0.3.1](https://github.com/jonkristian/alko/releases/tag/v0.3.1).
